pkg/ruler/base/notifier.go

```diff
@@ -88,10 +76,46 @@ func (rn *rulerNotifier) stop() {
 	rn.wg.Wait()
 }
 
+func getAlertmanagerTenantConfig(amConfig ruler_config.AlertManagerConfig, amOverrides ruler_config.AlertManagerConfig) (ruler_config.AlertManagerConfig, error) {
+	if amOverrides.AlertmanagerURL != "" {
```

Comment:
Why do we have to do this? Surely the zero value of `amConfig.AlertmanagerURL` is `""` anyway?

Reply:
exactly, so we should only use the override when we have a non zero value, hence the check

Reply:
Can't we just set the value, though? If the value is blank, then there's no override
Anyway, it's a nit - nothing serious

Reply:
well the problem is, if the global config sets a URL, but no value is set per tenant, the the override URL would be empty, which would erase the global URL in the config (unless I misunderstood something, this would not be the desired outcome)

Reply:
I see, thanks :+1: